docs/README.md

```diff
@@ -6,6 +6,7 @@
 
 ### Modules
 
+* ["@types/ethjs-util/index"](modules/__types_ethjs_util_index_.md)
```

Comment:
I couldn't find a way to ignore this file in typedoc without getting the error `Could not find a declaration file for module 'ethjs-util'.`, so I guess there is no harm in having it for now, although I wish the file was filled with the type info rather than just being empty 😅

The `library` mode in typedoc is pretty new and works quite well for our purposes, so perhaps it will be fixed or improved in the future:

```
Warning: File /Users/ryanghods/dev/ethereumjs-util/src/@types/ethjs-util/index.ts
is not a module and cannot be converted in library mode
```